### PR TITLE
Disable LIBCXX_INCLUDE_BENCHMARKS in Fuchsia cmake

### PR DESCRIPTION
**Pull request by Caslyn**
This disables `LIBCXX_INCLUDE_BENCHMARKS` on Fuchsia-Linux while dubugging Cmake configurations.

### CODE REVIEW

File: clang/cmake/caches/Fuchsia-stage2.cmake [hunk 1]
```diff
@@ -31,6 +31,7 @@ set(LLVM_STATIC_LINK_CXX_STDLIB ON CACHE BOOL "")
 set(LLVM_USE_RELATIVE_PATHS_IN_FILES ON CACHE BOOL "")
 set(LLDB_ENABLE_CURSES OFF CACHE BOOL "")
 set(LLDB_ENABLE_LIBEDIT OFF CACHE BOOL "")
+set(LIBCXX_INCLUDE_BENCHMARKS OFF CACHE BOOL "")
```
Comment (petrhosek) [inline clang/cmake/caches/Fuchsia-stage2.cmake:34]: This needs to be set for the target, that is line 184 for Linux.

Comment (Caslyn) [reply clang/cmake/caches/Fuchsia-stage2.cmake]: Thanks - Done. I looked at the pattern of where we set fellow `LIBCXX_...` macros and made adjustments in in Fuchsia.cmake as well.

Comment (Caslyn) [reply clang/cmake/caches/Fuchsia-stage2.cmake]: Also - PTAL @ https://github.com/llvm/llvm-project/pull/115361#issuecomment-2480526628.

**Comment by Caslyn** [comment]
Closing - this is resolved by https://github.com/llvm/llvm-project/pull/116644

**Pull request closed without merging**